### PR TITLE
quick view page initial

### PR DESCRIPTION
**Pull request by amnonkhen**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -11,12 +11,14 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
...
 import {ReactiveFormsModule} from "@angular/forms";
+import {QuickviewComponent} from "./grid/quickview.component";
 
 @NgModule({
   declarations: [
     AppComponent,
     GridComponent,
-    UrlCellRenderer
+    UrlCellRenderer,
+    QuickviewComponent
```
Comment (amnonkhen) [inline src/app/app.module.ts:21]: Please keep the lists here sorted for smoother git merges.

File: src/app/grid/grid.component.ts [hunk 2]
```diff
@@ -79,6 +80,12 @@ export class GridComponent implements OnInit  {
     toolPanels: ["filters", "columns"],
   };
 
+  onRowClick() {
+    console.log("Row clicked" );
+    this.router.navigate(['/quickview-component']);
+    //window.open("/quickview-component");
```
Comment (amnonkhen) [inline src/app/grid/grid.component.ts:86]: please remove commented code

File: src/app/grid/grid.component.ts [hunk 3]
```diff
@@ -79,6 +80,12 @@ export class GridComponent implements OnInit  {
     toolPanels: ["filters", "columns"],
   };
 
+  onRowClick() {
+    console.log("Row clicked" );
```
Comment (amnonkhen) [inline src/app/grid/grid.component.ts:84]: Please remove unnecessary logging from the console.

File: src/app/grid/quickview.component.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  height: 100%;
+}
+
+#myGrid {
+  flex: 1 1 0px;
+  width: 100%;
+}
+
+.quickview-wrapper {
+  width: 300px;
```
Comment (amnonkhen) [inline src/app/grid/quickview.component.scss:14]: I would prefer we used relative sizes in styles to  facilitate responsiveness.

File: src/app/grid/quickview.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import {QuickviewComponent} from "./quickview.component";
+
+describe('QuickviewComponent', () => {
```
Comment (amnonkhen) [inline src/app/grid/quickview.component.spec.ts:5]: Ideally here you would test that the component contains the correct information. There should also be a test that verifies the correct UI behaviour, i.e. when the user clicks the detail page shows.

**Comment by theisuru** [comment]
I treid to run it locally but detailed page was not openning, maybe I missed the link to open it

**Pull request closed without merging**